### PR TITLE
Update deps and projects structure for nerves v0.9

### PR DESCRIPTION
**Pull request by mobileoverlord**
Updates deps and project structure for nerves v0.9

I recommend updating nerves_bootstrap and fwup in your environment
You can update `nerves_bootstrap` by calling `mix local.nerves`
 and if you are on a mac you can update fwup with `brew upgrade fwup`

### CODE REVIEW

File: apps/fw_rpi/config/rpi/fwup.conf [hunk 1]
```diff
@@ -1,59 +1,93 @@
...
+# All of these can be overriden using environment variables of the same name.
+#
+#  Run 'fwup -m' to query values in a .fw file.
+#  Use 'fw_printenv' to query values on the target.
+#
+# These are used by Nerves libraries to introspect.
+define(NERVES_FW_PRODUCT, "Nerves Firmware")
+define(NERVES_FW_DESCRIPTION, "")
+define(NERVES_FW_VERSION, "${NERVES_SDK_VERSION}")
+define(NERVES_FW_PLATFORM, "rpi3")
```
Comment (mauricionr) [inline apps/fw_rpi/config/rpi/fwup.conf:18]: The older versions also will work?

**Review by mauricionr — COMMENTED** [review]
:rocket:

File: apps/fw_rpi/config/rpi/fwup.conf [hunk 2]
```diff
@@ -1,59 +1,93 @@
-# Firmware configuration file for the Raspberry Pi
+# Firmware configuration file for the Raspberry Pi 3
+
+require-fwup-version="0.15.0"  # For the trim() call
+
+#
+# Firmware metadata
+#
+
+# All of these can be overriden using environment variables of the same name.
```
Comment (mauricionr) [inline apps/fw_rpi/config/rpi/fwup.conf:9]: got it

**Comment by amolenaar** [comment]
Thanks. I'll check why the tests fail (SystemRegistry.TermStorage undefined).

**Comment by mauricionr** [comment]
i could build the firmware, but i'm also getting this error `SystemRegistry.TermStorage undefined`

**Comment by mauricionr** [comment]
i'm testing this branch https://github.com/mobileoverlord/nerves_dnssd/tree/system_registry

**Comment by mauricionr** [comment]
fixed in `system_registry` branch

**Comment by amolenaar** [comment]
Thanks Justin ans Mauricio. I’ll merge it ASAP. And build a new release of nerves_dnssd.

**Pull request merged**